src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "What's coming to Chrome 84 in the Web Animations API"
+subhead: Wrangling your web animations is about to get much easier
```

Comment:
```suggestion
title: Web Animations API improvements in Chromium 84
subhead: Orchestrating animations with promises, performance improvements with replaceable animations, smoother animations with composite modes, and more.
```

* Presumably these changes will also land in Microsoft Edge, in which case we should use `Chromium` throughout the article rather than `Chrome`. My proposed rephrasing of the title also provides more of a hint that there is a lot of new goodness coming (and it's more concise).
* The current `subhead` doesn't give me any hint as to what has changed. I want the subtitle to quickly inform users about what has changed so that they can decide whether to invest more time by reading the article.

Reply:
👍